charts/operator-wandb/charts/parquet/values.yaml

```
@@ -17,6 +17,9 @@ extraCors: []
 
 common: {}
 deployment: {}
+cronJob:
+  enabled: true 
+  schedule: "0 * * * *"
```

```suggestion
  schedule: "11 * * * *"
```
Arbitrary number so that it's not all at the strike of the hour.